pkg/cloudprovider/vm.go

```diff
@@ -14,8 +15,15 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	getVmOperationName = "vm_get"
+	putVmOperationName = "vm_create_or_update"
+)
+
+// VMClient client for VirtualMachines
 type VMClient struct {
-	client compute.VirtualMachinesClient
+	client   compute.VirtualMachinesClient
+	reporter *metrics.Reporter
```

Comment:
Cant we have a static declaration of metrics and call it directly from all the places without having to initialize for every package ?

Reply:
yes, we can think of something like  `metrics.Reporter` which can be initialized via `init()` in metrics package. All other packages can use directly metrics.Reporter to report any metric.
Will remove reference of `reporter *metrics.Reporter` from the structs like `VMClient /server` and others